src/ngrx/6-test-actions/test-actions.md

```diff
@@ -28,11 +28,85 @@ ng test --watch
 The `--watch` switch will rerun your tests whenever a code file changes. You can skip it to just run all tests once.
 
 
+## Introduction
+
+Each implementation section will be paired with a testing section. These testing sections will go over the basics of how to test the implementation and use of NgRx. Before continuing, you should have an intermediate understanding of the following:
+
+1. [Angular TestBeds](https://angular.io/guide/testing-utility-apis) - Although `TestBeds` aren't required for testing Angular applications, and [there are ways to test NgRx without a TestBed](https://ngrx.io/guide/store/testing#testing-without-testbed), we use `TestBeds` throughout this course.
+
+2. [Jasmine Unit Tests](https://github.com/jasmine/jasmine#a-javascript-testing-framework) - Throughout this course, we will be using [Jasmine to test our application](https://jasmine.github.io/tutorials/your_first_suite). Although the syntax will differ slightly between different testing tools such as [Mocha](https://mochajs.org/) and [Jest](https://jestjs.io/), the concepts used throughout this course will apply to whatever solution you use in your future projects.
+
+
 ## Description
 
-TODO: remove or use parts of for an intro for the section
+In this section will write unit tests involving Actions. When testing Actions, we don't typically test Actions directly. Instead, we test their use in Components, Effects, Reducers.
```

Comment:
in this section `you` will write (missing you?)

Reply:
or we

Reply:
done~